collectors/feature/gitlab/src/main/java/com/capitalone/dashboard/collector/FeatureSettings.java

```diff
@@ -0,0 +1,59 @@
+package com.capitalone.dashboard.collector;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * Bean to hold settings specific to the Feature collector.
+ * 
+ */
+@Component
+@ConfigurationProperties(prefix = "gitlab")
```

Comment:
I'm not familiar enough with how spring handles the properties stuff, I can play around with it this afternoon and see what happens.  But I'm cool to change it to whatever you guys think.